_dashboards/discover/index-discover.md

```diff
@@ -1,104 +1,115 @@
-2. **Analyze data**. You can analyze data, view individual documents, and create tables summarizing data contents.
-3. **Visualize data**. You can display findings from your saved searches in a single dashboard that combines different data visualization types.
+{::nomarkdown}<img src="{{site.url}}{{site.baseurl}}/images/icons/alert-icon.png" class="inline-icon" alt="alert icon"/>{:/} The legacy **Discover** tool will be retired in OpenSearch 2.11. The updated **Discover** tool provides you with new features and enhancements that make the data exploration experience in OpenSearch Dashboards more intuitive and user-friendly. To experience the updated tool now, install the [latest version of OpenSearch Dashboards](https://opensearch.org/downloads.html) or try out these features using the [OpenSearch Playground](https://playground.opensearch.org/app/home#/).  
+{: .note} 
 
-## Try it: Exploring sample data with Discover
+In this tutorial, you'll learn about using **Discover** to:
 
-This tutorial shows you how to use Discover to analyze and understand a sample dataset. At the end of this tutorial, you should be ready to use Discover with your own data.
+- Add data
+- Explore and visualize data
+- Share the data findings
+- Set alerts
 
-Before starting this tutorial, make sure you've added the **Sample flight data**. See [Quickstart guide for OpenSearch Dashboards]({{site.url}}{{site.baseurl}}/dashboards/quickstart-dashboards/) for information about how to get started.
-{: .warning}
+#### Prerequisites
 
-### Setting up data
+The following are prerequisites for using **Discover** effectively:
 
-Watch the following short video or start with the tutorial steps to learn how to set up a sample dataset in Discover.
+- Install [OpenSearch Dashboards 2.10 or later](https://opensearch.org/downloads.html).
+- Add OpenSearch [sample data]({{site.url}}{{site.baseurl}}/dashboards/quickstart/) or import your own data into OpenSearch.
+- Have foundational understanding of OpenSearch [documents and indexes]({{site.url}}{{site.baseurl}}/im-plugin/index/).   
 
-![Setting up the sample data in Discover]({{site.url}}{{site.baseurl}}/images/discover-setting-up-data.gif)
+## Import data
 
-1. Verify access to OpenSearch Dashboards by connecting to [http://localhost:5601](http://localhost:5601) from a browser. The default username and password are `admin`. 
-1. On the **Home** page, choose **Discover** in the navigation pane.
-1. On the index pattern toolbar, select the **opensearch_dashboards_sample_data_flights** dataset.
-1. On the time filter toolbar, choose the calendar icon and then change the time range to **Last 7 days**.
+You must add data to OpenSearch before you can start exploring it. This tutorial uses sample data. To add the sample data, follow these steps:
 
-### Exploring the data fields
+1. On the OpenSearch Dashboards Home page, choose **Add sample data**.
```

Comment:
Is "home" intentionally capitalized here?

Reply:
No, that was my oversight.